canonical-web-and-design/charmhub.io
# 749 bundle overview page

## Done
- _Update bundle icon on home page_
- _Update bundle icon on search page_
-  _Add svg topology to bundle overview page_
- _Show only the `Overview` tab for bundles_

## How to QA
- Visit the demo in your browser, link commented below
  - Or you can run the project locally using the [dotrun](https://snapcraft.io/dotrun) snap with `$ dotrun` and view it in your web browser at: http://localhost:8045/
  - If your demo doesn't work, read more about the [demoservice](https://discourse.ubuntu.com/t/demoservice/16862)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Note that bundles are displayed and the icon looks right
- Click on one bundle and see the topology on the overview page
- Search for `kubeflow` and see the search results look right
- Go to `/prometheus` and open the console, and see there are no errors

## Issue / Card
Fixes 

## Screenshots
[if relevant, include a screenshot]
